.gitignore

```diff
@@ -13,4 +13,4 @@ coverage.html
 
 perf/perf
 pulsar-perf
-bin
+bin
```

Comment:
Please do not remove the line break at the end of a file. If you're using the JetBrains' IDEs, you might do that by selecting **Editer - General - Ensure every saved file ends with a line break**.

https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
Fixed, thanks.